include/swift/Runtime/Metadata.h

```diff
@@ -2137,13 +2137,13 @@ struct ProtocolConformanceRecord {
   
   // The conformance, or a generator function for the conformance.
   union {
-    /// A direct reference to the witness table for the conformance.
-    RelativeDirectPointer<WitnessTable> WitnessTable;
+    /// A reference to the witness table for the conformance.
+    RelativeIndirectablePointer<WitnessTable> WitnessTable;
     
     /// A function that produces the witness table given an instance of the
     /// type. The function may return null if a specific instance does not
     /// conform to the protocol.
-    RelativeDirectPointer<WitnessTableAccessorFn> WitnessTableAccessor;
+    RelativeIndirectablePointer<WitnessTableAccessorFn> WitnessTableAccessor;
```

Comment:
OK, will try first what @hpux735 suggested and see if it still needs to be done.